client/src/document/hooks.ts

```diff
@@ -75,27 +79,24 @@ function prevHeading(heading: Element) {
 function addBreakoutButton(
   element: Element | null,
   id: string,
-  doc: Doc,
   code: EditorContent,
   locale: string
 ) {
   if (!element || element.querySelector(".play-button")) return;
-  const button = document.createElement("button");
-
-  button.textContent = "Play";
-
-  button.setAttribute("type", "button");
-  button.setAttribute("class", "play-button external");
-  button.title = "Open in Playground";
-  element.appendChild(button);
-
-  button.onclick = async () => {
-    const key = `play-${id}-${doc.mdn_url}`;
-    sessionStorage.setItem(key, JSON.stringify(code));
-    const url = new URL(window?.location.href);
-    url.pathname = `/${locale}/play`;
-    url.searchParams.set("sample", key);
-    window.location.href = url.href;
+  const a = document.createElement("a");
+
+  a.textContent = "Play";
+
+  a.setAttribute("class", "play-button external");
+  a.href = `/${locale}/play`;
+  a.setAttribute("data-play", id);
+  a.title = "Open in Playground";
+  element.appendChild(a);
+
+  a.onclick = async (e) => {
```

Comment:
Any reason we use `onclick` rather than `addEventListener`?
Same for `async` (from what I see there's no async code within)?